CHANGELOG.md

```diff
@@ -171,7 +177,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 * Pager plugin.
 * Progress plugin.
 
-[unreleased]: https://github.com/qgrid/ng2/compare/v5.3.3...HEAD
+[unreleased]: https://github.com/qgrid/ng2/compare/v5.3.4...HEAD
+[5.3.4]: https://github.com/qgrid/ng2/compare/v5.3.4...v5.4.2
```

Comment:
It's classic by me: comparison links for 5.3.4 and 5.3.3 versions are incorrect :)